crates/xxi-node/src/commons/message.rs

```diff
@@ -41,6 +41,11 @@ pub enum Message {
         order_id: Uuid,
         error: TradingError,
     },
+    PaymentReceived {
```

Comment:
🙃 Maybe we should talk about Lightning payment here (and in other places) to disambiguate.

Reply:
I renamed it to `LnPaymentReceived`